crates/bin/prove_block/tests/prove_block.rs

```diff
@@ -0,0 +1,35 @@
+use cairo_vm::types::layout_name::LayoutName;
+use cairo_vm::vm::errors::cairo_run_errors::CairoRunError;
+use prove_block::{prove_block, ProveBlockError};
+use rstest::rstest;
+use starknet_os::error::SnOsError;
+
+// # These blocks verify the following issues:
+// # * 76793: the first block that we managed to prove, only has a few invoke txs
+// # * 76766 / 76775: additional basic blocks
+// # * 86507 / 124533: a failing assert that happened because we used the wrong VersionedConstants
+// # * 87019: diff assert values in contract subcall
+#[rstest(
+    block_number => [76793, 76766, 76775, 86507, 87019, 124533]
+)]
```

Comment:
Can you use the `case` syntax for this? This way you can add some doc to each test like this:

```
#[case::small_block_with_only_invoke_txs(76793)]
#[case::fix_diff_assert_values_in_contract_subcall(87019)]
```